lib/auth/helpers.go

```diff
@@ -377,9 +378,33 @@ func (a *TestAuthServer) GenerateUserCert(key []byte, username string, ttl time.
 	return certs.ssh, nil
 }
 
+func keypairToPublicKeys(privateKey, publicKey []byte) (tlsPublicKey, sshPublicKey []byte, err error) {
+	sshPrivate, err := ssh.ParseRawPrivateKey(privateKey)
+	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
+
+	tlsPublicKey, err = tlsca.MarshalPublicKeyFromPrivateKeyPEM(sshPrivate)
+	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
+
+	return tlsPublicKey, publicKey, nil
```

Comment:
It looks like publicKey enters and leaves unchanged? Drop both the parameter and the return?